.github/workflows/build_and_test.yml

```diff
@@ -1,3 +1,5 @@
+## me fail english? that's unpossible.
+
```

Comment:
fix not maintainable 

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
looks good now- bravo

---
**Sema Reaction:** :ok_hand: This code looks good

Reply:
not maintainable

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix